src/Data/DefaultDataProviderDBalUtils.php

```diff
@@ -100,6 +101,10 @@ public static function addSorting(ConfigInterface $config, QueryBuilder $queryBu
         }
 
         foreach ($config->getSorting() as $sort => $order) {
+            if (empty($sort)) {
```

Comment:
Why test for empty `$sort`. Can you explain that.

Reply:
frag den Meister @discordier ;-)

Reply:
@discordier can you explain that? The key of an foreach loop can not be should empty.

Reply:
If so, should a warning be triggered?

Reply:
In addition to the exception or instead?
If we opt for exception, no warning, if we opt for ignoring, we should trigger a warning.

Reply:
I would be in favour of ignoring that and trigger the warning.